index.html

```diff
@@ -54,5 +54,147 @@ <h1 id="greeting">I’m Andrew. Glad to see you!</h1>
         </li>
       </ul>
     </section>
-  </body>
+    <ul id="works">
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio1.png" alt="" />
+        <p class="projectName_holder">Tonic</p>
+
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio2.png" alt="" />
+        <p class="projectName_holder">Multi-Post Stories</p>
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio3.png" alt="" />
+        <p class="projectName_holder">Tonic</p>
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+      <li class="card">
+        <img class="card_image" src="Snapshoot Portfolio4.png" alt="" />
+        <p class="projectName_holder">Multi-Post Stories</p>
+        <ul class="canopy">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="service">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="projectBTN"><button>See Project</button></div>
+      </li>
+      
+    </ul>
+      <section id="about">
```

Comment:
- [ ] Looking at the screenshot below, you can see that the project cards and the works section have different background colors, I'll suggest you implement this. :+1: :+1: 
- [ ] Also you can see you that the About section have a border-radius at the top right corner. I'll suggest you implement this too.

- - Figma design:
![Selection_079](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/107632780/54ff1bd9-ba6e-49cc-8c54-8f6dbb2c1748)

- - Your design:
![Screenshot from 2023-07-07 09-55-37](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/107632780/27396b73-3243-431c-a94c-6f9ef103d3a9)